### PR TITLE
User model

### PR DESCRIPTION
**Pull request by srwilliamg**
## Summary

- A new migration that creates the user table was added
- The user model was added to models directory

## Trello Card

https://trello.com/c/rsxwu7dw/21-modelo-de-usuario

### CODE REVIEW

File: migrations/migrations/20210104142345-add-user-table.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+'use strict';
```
Comment (blueJayWays) [inline migrations/migrations/20210104142345-add-user-table.js]: Is this necessary?

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/* eslint-disable require-atomic-updates */
+const bcrypt = require('bcrypt');
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
```
Comment (blueJayWays) [inline app/models/user.js]: for convention, model's name must be singular.

File: app/models/user.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            args: true,
+            msg: 'Name cannot be null'
+          },
+          notEmpty: {
+            args: true,
+            msg: 'Name cannot be empty'
+          }
+        }
+      },
+      last_name: { type: DataTypes.STRING },
```
Comment (blueJayWays) [inline app/models/user.js]: camelCase in the models. snake_case in the migrations.

File: app/models/user.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        validate: {
+          len: {
+            args: [8, 60],
+            msg: 'Password length should be between 8 and 60'
+          }
+        }
+      }
+    },
+    {
+      timestamps: false,
```
Comment (blueJayWays) [inline app/models/user.js]: you should mark this option as `true`. most probably you need to know when a user was created or modified.

File: migrations/migrations/20210104142345-add-user-table.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      last_name: { type: Sequelize.STRING, allowNull: false },
+      email: {
+        type: Sequelize.STRING,
+        unique: true,
+        allowNull: false
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false
+      }
```
Comment (blueJayWays) [inline migrations/migrations/20210104142345-add-user-table.js:29]: if timestamps is true in the model, so you need to add these fields:  updated_at and created_at

**Review by blueJayWays — CHANGES_REQUESTED** [review]

File: migrations/migrations/20210104142345-add-user-table.js (hunk 1 above)
Comment (srwilliamg) [reply migrations/migrations/20210104142345-add-user-table.js]: it's habit, I'm going to remove it

File: app/models/user.js (hunk 2 above)
Comment (srwilliamg) [reply app/models/user.js]: Changed

File: app/models/user.js (hunk 3 above)
Comment (srwilliamg) [reply app/models/user.js]: done :)

File: app/models/user.js (hunk 4 above)
Comment (srwilliamg) [reply app/models/user.js]: done :)

File: migrations/migrations/20210104142345-add-user-table.js (hunk 5 above)
Comment (srwilliamg) [reply migrations/migrations/20210104142345-add-user-table.js:29]: done :)

**Pull request merged**